docs/dunst.pod

```diff
@@ -264,6 +264,15 @@ The progress value can be set with a hint, too.
 
 =back
 
+=head1 ACTIONS
+
+Dunst allows interaction with actions in notifications. If "show_indicators" is
+true, a "(A)" will be appended to the notification title. The "context"
```

Comment:
> If "show_indicators" is true, a "(A)" will be appended to the notification title

`(A)` will be *prepended* not *appended* and not to the title, it will be prepended to the format.

This sentence also doesn't answer *when* it will be prepended.
> If one or more actions are available "(A)" will be prepended to the notification text